fastlane/Fastfile

```diff
@@ -158,18 +170,19 @@ platform :android do
     UI.user_error!("Release branch for version #{base_version} doesn't exist.") unless checkout_success
 
     # Check versions
+    version_code ||= build_code_next # Only compute this version_code fallback _after_ the checkout of the release branch
     message = <<-MESSAGE
 
       Current beta version: #{beta_version_current}
       New beta version: #{beta_version_next}
 
       Current build code: #{build_code_current}
-      New build code: #{options[:version_code] || build_code_next}
+      New build code: #{version_code}
 
     MESSAGE
 
     UI.important(message)
-    unless options[:skip_confirm] || UI.confirm('Do you want to continue?')
+    unless skip_confirm || UI.confirm('Do you want to continue?')
```

Comment:
<table data-meta="generated_by_pr-check">
  <tbody>
    <tr>
      <td>:no_entry_sign:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/Automattic/pocket-casts-android/blob/6a0928aa27197a8a7fec52a5f2f55ad056cabff3/fastlane/Fastfile#L185"></span>Style/IfUnlessModifier: Favor modifier <code>unless</code> usage when having a single-line body. Another good alternative is the usage of control flow <code>&amp;&amp;</code>/<code>||</code>.</td>
    </tr>
  </tbody>
</table>